.env

```diff
@@ -1 +1 @@
-# HF_TOKEN=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
+KISSKI_API_KEY='b6c89c0b03ba5bd30170933cc6861886'
```

Comment:
Hi Yixin,

never store API keys on Github.

I recommed you to revoke the huggingface and the Kisski key. Otherwise others can use the platforms on your behalf.

https://docs.github.com/en/rest/authentication/keeping-your-api-credentials-secure?apiVersion=2022-11-28#store-your-authentication-credentials-securely

Best,
Robert

Reply:
Hi Robert,

Thanks for your advice. I got a message last night, and in my latest version that I replaced it as "KISSKI_API_KEY=your_api_key" and uploaded it yesterday, but I don't know why you can still read it.

Best,
Yixin

Reply:
Because the git history contains it. Again, I recommend to revoke all keys you put on the internet.

Reply:
Do you know how to delete them? I tried using the "delete file" function, but it does not seem to work.

Reply:
You should revoke the key instead of deleting the history.

Reply:
E.g. as shown here: https://huggingface.co/docs/hub/en/security-tokens#how-to-manage-user-access-tokens or by sending an email to the KISSKI folks

Reply:
They have already created a new one and the HF token has already been deleted.